benchmarks/bench.rb

```diff
@@ -1,41 +0,0 @@
-require 'benchmark/ips'
```

Comment:
I'd like to keep this benchmark (added in https://github.com/ruby/json/pull/580) as it's very simple (e.g. it doesn't need to run under Bundler) and is a convenient way to compare the pure-Ruby backend and the extension.
Do you mind if I add it back under `benchmark`?

Reply:
Alright, I just moved it: 7b688009915aea4cc8e050843cad8446a0d739e3

Reply:
Thanks!